src/hotspot/share/gc/shenandoah/shenandoahReferenceProcessor.hpp

```diff
@@ -82,13 +82,11 @@ class ShenandoahRefProcThreadLocal : public CHeapObj<mtGC> {
   Counters _encountered_count;
   Counters _discovered_count;
   Counters _enqueued_count;
+  NONCOPYABLE(ShenandoahRefProcThreadLocal);
```

Comment:
I think this change should not be made.  We Oracle folks don't make casual changes to Shenandoah, since we don't support or test it.  The Shenandoah team is free to write it out rather than using the macro if they prefer.

Reply:
OTOH, it is fine to suggest minor/cosmetic changes to the entirety of the OpenJDK code, and ask others to review/test the bits that are not built/tested by the submitter. 

This change is fine (AFAICS, it is semantically the same as the old code), and it passes `hotspot_gc_shenandoah`.

Reply:
Then I'll withdraw my objection.